packages/bundle/src/serialized.ts

```diff
@@ -43,9 +43,11 @@ type SerializedTLogEntry = {
       }
     | undefined;
   integratedTime: string;
-  inclusionPromise: {
-    signedEntryTimestamp: string;
-  };
+  inclusionPromise:
+    | {
+        signedEntryTimestamp: string;
+      }
+    | undefined;
```

Comment:
Updating the serialized form of the bundle to account for the fact that `v0.2` bundles may have an empty `inclusionPromise`